packages/grid/_modules_/grid/models/gridSlotsComponent.ts

```diff
@@ -73,7 +73,7 @@ export interface GridSlotsComponent extends GridIconSlotsComponent {
   /**
    * Pagination component rendered in the grid footer by default.
    */
-  Pagination?: React.ElementType<GridBaseComponentProps & React.HTMLAttributes<HTMLDivElement>>;
+  Pagination?: React.ElementType<GridBaseComponentProps>;
```

Comment:
Why this change of type?

Reply:
I reverted it back to the original since it doesn't seem to impact the user. That way can revert the change I did in the storybook customization example for the pagination. The build was failing because the type was not alowed.